activestorage/lib/active_storage/service/disk_service.rb

```diff
@@ -160,7 +160,7 @@ def url_helpers
       end
 
       def url_options
-        ActiveStorage::Current.url_options
+        ActiveStorage::Current.url_options || Rails.application.routes.default_url_options
```

Comment:
Active Storage should not depend on the Rails application. Maybe we should inject it into the DiskService using the Engine in Active Storage?

Reply:
https://github.com/rails/rails/pull/42847/commits/3b0556e135ebc4685896d1e0e2ef9fad5370cfdb, It is what you mean? Let me know what you think.